src/CheckoutSdk/Common/CustomerRequest.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Checkout.Payments
+﻿namespace Checkout.Common
```

Comment:
This is a breaking change. I would recommend creating a specific customer request/response class specifically for sources or referencing the payments one for now until we can normalise these later.